File: app/src/main/java/com/pixels/bdpbiblio/libro.java
package com.pixels.bdpbiblio;

public class libro
{
	String Codigo,Titulo,Edicion,Ciudad,Anno,Editorial,Descripcion,Valorl,Nombres,Apellidos;
	public libro(String codigo,String titulo,String edicion,String ciudad,String anno,String editorial,String descripcion,String valorl,String nombres,String apellidos){
		this.Codigo=codigo;
		this.Titulo=titulo;
		this.Edicion=edicion;
		this.Ciudad=ciudad;
		this.Anno=anno;
		this.Editorial=editorial;
		this.Descripcion=descripcion;
		this.Valorl=valorl;
		this.Nombres=nombres;
		this.Apellidos=apellidos;
	}
	public String getCodigo(){
		return Codigo;
	}
	public String getTitulo(){
		return Titulo;
	}
	public String getEdicion(){
		return Edicion;
	}
	public String getCiudad(){
		return Ciudad;
	}
	public String getAnno(){
		return Anno;
	}
	public String getEditorial(){
		return Editorial;
	}
	public String getDescripcion(){
		return Descripcion;
	}
	public String getValorl(){
		return Valorl;
	}
	public String getNombres(){
		return Nombres;
	}
	
	public String getApellidos(){
		return Apellidos;
	}

}

File: webservice/libro.php
<?php




$host = "127.0.0.1";




$port = "5432";




$user = "postgres";




$database="biblioteca";




$password="67895421d";




$conexion=pg_connect("host=$host port=$port user=$user dbname=$database password=$password");





//si fallara la conexion con la BD 





if (!$conexion) { 




echo "error en la conexion"; 




}




else




{ 



$codigo=$_GET['codigo'];


//$nombre=$_GET['nombre'];



//echo $nombre;




$query = "select libro.codigo,titulo,edicion,ciudad,anno,editorial,descripcion,valorl,nombres,apellidos from (((librosautores inner join libro on librosautores.codigo=libro.codigo) inner join autor on librosautores.codigoa=autor.codigo) inner join persona on autor.codigo=persona.codigo) where libro.codigo='$codigo'";



$consulta = pg_query($conexion, $query);




if(!$consulta){




}else{




while($row = pg_fetch_array($consulta)){




$flag[] =$row; 




} 




print(json_encode($flag));




}




pg_close($conexion);




}





?>



